_blog-src/_posts/2015-07-07-fuse-apis.markdown

```diff
@@ -0,0 +1,171 @@
+
+![APIs deployed to Fuse Fabric](/blog/images/2015-07-07/apis.png)
+
+There is a default user preconfigured so you can verify if the application was successfully deployed. It's available at `<location>/customerservice/customers/123` (e.g. [http://localhost:8182/cxf/crm/customerservice/customers/123](http://localhost:8182/cxf/crm/customerservice/customers/123))
+
+{% highlight xml %}
+[sbunciak@sbunciak ~]$ http http://localhost:8182/cxf/crm/customerservice/customers/123
+HTTP/1.1 200 OK
+Content-Type: application/xml
+Date: Tue, 07 Jul 2015 22:17:13 GMT
+Server: Jetty(8.1.17.v20150415)
+Transfer-Encoding: chunked
+
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ns0:Customer xmlns:ns0="http://rest.fabric.quickstarts.fabric8.io/">
+    <ns0:id>123</ns0:id>
+    <ns0:name>John</ns0:name>
+</ns0:Customer>
+{% endhighlight %}
+
+Configure apiman to manage Fuse endpoint
+========================================
+
+To install the latest version of apiman follow this [guide](http://www.apiman.io/latest/download.html). The installation process is again very simple, all you need to do is to extract apiman overlay zip on top of [Wildly 8](http://www.wildfly.org/) server. 
+
+After completing the installation, start apiman by executing the follogin command from `<wildfly>/bin` folder:
+
+{% highlight sh %}
+./standalone.sh -c standalone-apiman.xml
+{% endhighlight %}
+
+### Quick public service setup
+
+For demonstration purposes I'll be creating a Public Service, however in real life you might want to configure different plans, various contracts, etc. 
+Please consult the apiman [user guide](http://www.apiman.io/latest/user-guide.html) for more details.
+
+In order to manage APIs in apiman you need to create a new Organization to which your APIs will belong. 
+You can do that easily in apiman manager, typically available at: [http://localhost:8080/apimanui](http://localhost:8080/apimanui).
+
+Once logged in apiman manager, locate the Organizations sections on the initial page, select _Create a New Organization_ link, provide a name and hit _Create Organization_ button. This will take you to organization details page where you can create a new Service by clicking on _New Service_ button under _Services_ tab:
+
+![Image: Create Service](/blog/images/2015-07-07/service.png)
+
+You will be asked to provide a name and a version for this Service. Once the service is successfully created there are few things remaining before you can publish and start using it. 
+First, you need to provide an implementation base url on the _Implementation_ tab. Use the base URL of the REST Quickstart and save your changes:
+
+![Image: Provide Endpoint Implementation](/blog/images/2015-07-07/implementation.png)
+
+Second, you may want to apply some policies to this Publis Service. To do that, go to Policies tab and configure a policy of your choice. 
```

Comment:
- Publis Service
- Public Service

- go to Policies tab
- go to the Policies tab